### PR TITLE
Update event marketing rolebook

### PR DESCRIPTION
**Pull request by mrbobbytables**
ref: #4374 

Fixed formatting and added guidelines regarding access to both contributors@kubernetes.io and the CNCF/kubernetes twitter handles.

/hold
for review

/assign @parispittman @jonasrosland @castrojo

### CODE REVIEW

**Comment by mrbobbytables** [comment]
Noticed one thing that should be fixed 👍

File: events/events-team/marketing/README.md [hunk 1]
```diff
@@ -26,16 +31,59 @@ As Marketing Lead, you are responsible for the overall marketing communications
 - Create a communication schedule - what, when, how, who  
 - Manage online presence - website, social media strategy  
 - Potentially recruit a social media coordinator role  
-- Provide updates on social media, mailing lists, and Slack throughout pre-, during-, and post-event  
-- Assist with GitHub event repo adds and edits  
-- Work with Event Lead on a recap blog  
+- Provide updates on social media, mailing lists, and Slack throughout pre-,
+  during-, and post-event
+- Assist with GitHub event repo adds and edits
+- Work with Event Lead on a recap blog
```
Comment (jeefy) [inline events/events-team/marketing/README.md]: Technically this should be more broad. 

"... on any necessary blog posts (such as the schedule announcement and recap blog posts)"

File: events/events-team/marketing/README.md [hunk 2]
```diff
@@ -26,16 +31,59 @@ As Marketing Lead, you are responsible for the overall marketing communications
...
 
+### Communication Channels and Property Access
+
+Communications to the Kubernetes Contributor Community should be sent from
+official Kubernetes accounts to ensure they reach the community. Mass messages
+from personal accounts can easily wind up being marked as spam.
+
+**Email**
+
+Mail should be sent from the contributors@kubernetes.io account. For access work
```
Comment (jeefy) [inline events/events-team/marketing/README.md]: nit: "For access**,** work with..."

File: events/events-team/marketing/README.md [hunk 3]
```diff
@@ -26,16 +31,59 @@ As Marketing Lead, you are responsible for the overall marketing communications
...
+Tweets can be sent from two accounts: [@K8sContributors] and [@Kubernetesio].
+
+Tweets targeting the greater Kubernetes community should use the [@Kubernetesio]
+account. Tweets using this account must be coordinated with the [CNCF]. Work
+with the CNCF events rep to schedule the tweets or put you in contact with
+the CNCF marketing members in charge of the account. Tweets using this account
+are often limited in the content and quantity that may be sent.
+
+Tweets targeting the current contributor base should use the [@K8sContributors]
+account. There are no restrictions with this account for the amount of kind of
```
Comment (jeefy) [inline events/events-team/marketing/README.md]: nit: amount **or** kind of messages

File: events/events-team/marketing/README.md (hunk 1 above)
Comment (mrbobbytables) [reply events/events-team/marketing/README.md]: Going to forgo updating that in this PR. The guide needs more work.

**Comment by mrbobbytables** [comment]
Comments addressed 👍

**Pull request merged**